src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/UpdateDashboardOnVacancyChange.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Esfa.Recruit.Vacancies.Client.Application.Services;
+using Esfa.Recruit.Vacancies.Client.Domain.Events;
+using Esfa.Recruit.Vacancies.Client.Domain.Projections;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.EventHandlers
+{
+    public class UpdateDashboardOnVacancyChange : INotificationHandler<VacancyCreatedEvent>,
+                                                    INotificationHandler<VacancyUpdatedEvent>,
+                                                    INotificationHandler<VacancySubmittedEvent>,
+                                                    INotificationHandler<VacancyDeletedEvent>
+    {
+        private readonly ICreateDashboards _dashboardService;
+        private readonly ILogger<UpdateDashboardOnVacancyChange> _logger;
+
+        public UpdateDashboardOnVacancyChange(ICreateDashboards dashboardService, ILogger<UpdateDashboardOnVacancyChange> logger)
+        {
+            _logger = logger;
+            _dashboardService = dashboardService;
+        }
+
+        public async Task Handle(VacancyCreatedEvent notification, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("Handling VacancyCreatedEvent for accountId: {employerAccountId}", notification?.EmployerAccountId);
+            await ReBuildDashboard(notification.EmployerAccountId);
+        }
+
+        public async Task Handle(VacancyUpdatedEvent notification, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("Handling VacancyUpdatedEvent for accountId: {employerAccountId}", notification?.EmployerAccountId);
```

Comment:
Can we use `notification.GetType().Name`? also are you missing the $ to do the string interpolation for `employerAccountId`

Reply:
It's not using string interpolation. The logger uses the string format and the place holder names as the property names. You'll see in them as properties on the log entry in kibana.

Reply:
👍

Reply:
Could do this: 

```$"Handling {notification.GetType().Name} for accountId: {{employerAccountId}}"```

What do you think?